lib/ui/src/modules/ui/components/routed_link.js

```diff
@@ -0,0 +1,44 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+// Cmd/Ctrl/Shift/Alt + Click should trigger default browser behaviour. Same applies to non-left clicks
+function isPlainLeftClick(e) {
+  return e.button === 0 && !e.altKey && !e.ctrlKey && !e.metaKey && !e.shiftKey;
+}
+
+export default class RoutedLink extends React.Component {
+  constructor(...args) {
+    super(...args);
+    this.onClick = this.onClick.bind(this);
+  }
+
+  onClick(e) {
+    if (this.props.onClick && isPlainLeftClick(e)) {
+      e.preventDefault();
+      this.props.onClick(e);
+    }
+  }
+
+  render() {
+    const { onClick, href, children, overrideParams, ...restProps } = this.props;
```

Comment:
Null is also a value. Try and pass it to some dom element, react warns you (and from version 16 it will silently append it as an attribute)